src/cpp/core/CMakeLists.txt

```diff
@@ -104,7 +104,6 @@ set (CORE_SOURCE_FILES
    r_util/RSessionContext.cpp
    r_util/RTokenizer.cpp
    r_util/RSourceIndex.cpp
-   r_util/RTokenizerTests.cpp
```

Comment:
Note that all files matching the glob `*Tests.cpp` are implicitly added to the test executables built so this entry was unnecessary.